includes/functions/llms.functions.certificate.php

```diff
@@ -107,7 +107,7 @@ function llms_get_certificate_content( $id = 0 ) {
  * }
  */
 function llms_get_certificate_fonts() {
-
+	$use_g_fonts = apply_filters( 'llms_use_google_webfonts', false );
```

Comment:
Needs a filter doc:

```suggestion
	/**
	 * Determines whether or not webfonts are loaded from Google CDNs.
	 *
	 * @since [version]
	 *
	 * @param bool $use_g_fonts If `true`, fonts are loaded from Google, otherwise they are loaded from the local site.
	 */
	$use_g_fonts = apply_filters( 'llms_use_google_webfonts', false );
```